pkg/services/ngalert/notifier/channels/testing.go

```diff
@@ -86,6 +86,10 @@ type emailSender struct {
 	ns *notifications.NotificationService
 }
 
+func (e emailSender) SendWebhook(ctx context.Context, cmd *channels.SendWebhookSettings) error {
+	panic("not implemented")
```

Comment:
Why panicking here? Wouldn't it be better to just return an error?

Reply:
Because this is the email stub for email sender.